654.75
Loading pipeline components...:   0%|          | 0/7 [00:00<?, ?it/s]Loading pipeline components...:  14%|█▍        | 1/7 [00:00<00:02,  2.06it/s]Loading pipeline components...:  29%|██▊       | 2/7 [00:00<00:02,  2.40it/s]Loading pipeline components...:  43%|████▎     | 3/7 [00:01<00:02,  1.85it/s]Loading pipeline components...:  86%|████████▌ | 6/7 [00:01<00:00,  4.86it/s]Loading pipeline components...: 100%|██████████| 7/7 [00:01<00:00,  4.06it/s]
Editing from 0!
Traceback (most recent call last):
  File "inference.py", line 57, in <module>
    free, total = torch.cuda.mem_get_info(torch.device('cuda'))
  File "/home/s1/eugenebaek/anaconda3/envs/diffusers_ip2p/lib/python3.8/site-packages/torch/cuda/memory.py", line 591, in mem_get_info
    device = _get_device_index(device)
  File "/home/s1/eugenebaek/anaconda3/envs/diffusers_ip2p/lib/python3.8/site-packages/torch/cuda/_utils.py", line 34, in _get_device_index
    return _torch_get_device_index(device, optional, allow_cpu)
  File "/home/s1/eugenebaek/anaconda3/envs/diffusers_ip2p/lib/python3.8/site-packages/torch/_utils.py", line 540, in _get_device_index
    raise ValueError('Expected a torch.device with a specified index '
ValueError: Expected a torch.device with a specified index or an integer, but got:cuda
